app/build.gradle

```diff
@@ -1,4 +1,18 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.dicedmelon.gradle:jacoco-android:0.1.4'
+    }
+}
+
```

Comment:
The above script should be in the project level `build.gradle` not in the app level. Kindly, fix that ..

Reply:
thanks for that feedback